src/utils/db.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require('sequelize');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const sequelize = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
Consider adding a `logging` option to the Sequelize configuration to control the amount of logging output. You can set it to `false` to disable logging or provide a custom function to handle logs.